### PR TITLE
Added description to package smart contract command

### LINKED ISSUE #1399: Provide greyed out helpers (dir paths) for workspace folders in Package command
I accidentally added two "contract" folders from commercial paper (different locations, one for DigiBank, one for MagnetoCorp) into my workspace. When I came to run the Package Command I got a screenshot that asked me whether I wanted to package "contract" or "contract". I guessed that the list was in the same order as the folders in my workspace. 

Could the Package command be updated to have some greyed out helpers after the folders to show their full paths? 

Jake knows what I mean. Might be relevant for other commands too?

image

### PR DESCRIPTION
**Pull request by toggle007**
Path of the contract now in grey so can distinguish the contracts (if they're named the same but have different paths).
#1399 
Signed-off-by: Akshat Shah <akshatshah@akshats-mbp.hursley.uk.ibm.com>

### CODE REVIEW

File: test/commands/UserInputUtil.test.ts [hunk 1]
```diff
@@ -898,6 +898,48 @@ describe('UserInputUtil', () => {
                 placeHolder: 'choose a folder'
             });
         });
+
+        it('should show the workspace folders along with their path as a description', async () => {
```
Comment (cazfletch) [inline test/commands/UserInputUtil.test.ts]: you don't need a separate test you can just add to the existing one as they both do the same thing

File: test/commands/UserInputUtil.test.ts [hunk 2]
```diff
@@ -898,6 +898,48 @@ describe('UserInputUtil', () => {
...
+            result.should.deep.equal({
+                label: 'myPath1', data: {
+                    name: 'myPath1',
+                    uri: {
+                        path: 'path1'
+                    }
+                }, description : 'pathHere'
+            });
+
+            quickPickStub.should.have.been.calledWith(sinon.match.any, {
```
Comment (cazfletch) [inline test/commands/UserInputUtil.test.ts]: you should change `sinon.match.any` to be the quick pick items that should be shown so you can test the it shows the correct description

File: test/commands/UserInputUtil.test.ts (hunk 1 above)
Comment (toggle007) [reply test/commands/UserInputUtil.test.ts]: Done

File: test/commands/UserInputUtil.test.ts (hunk 2 above)
Comment (toggle007) [reply test/commands/UserInputUtil.test.ts]: I believe I have now got the correct quick pick items - test passes.

**Pull request merged**